src/main/java/com/fasterxml/jackson/datatype/jsr310/ser/key/Jsr310NullKeySerializer.java

```diff
@@ -0,0 +1,33 @@
+package com.fasterxml.jackson.datatype.jsr310.ser.key;
+
+import java.io.IOException;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+/**
+ * This class is to be used in case {@code null} keys are needed to be serialized in a {@link Map} with Java 8 temporal keys. By default the
+ * {@code null} key is not supported by jackson, the serializer needs to be registered manually.
+ *
+ * @author Zoltan Kiss
+ * @since 2.6
```

Comment:
By default an exception is thrown when there are null keys. There might be a default null key serializer somewhere, I did not find it. Also I found it desirable to be in control of the null key format as it is important at deserialization time.